src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs

```diff
@@ -260,6 +260,85 @@ public bool Read()
             return retVal;
         }
 
+        /// <summary>
+        /// Skips the children of the current JSON token.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown when the reader was given partial data with more data to follow (i.e. _isFinalBlock is false).
+        /// </exception>
+        public void Skip()
+        {
+            if (!_isFinalBlock)
```

Comment:
Is there any way that a caller who receives an already open Utf8JsonReader can know if it's in this state?  I don't see a property exposing this value...

Reply:
> Is there any way that a caller who receives an already open Utf8JsonReader can know if it's in this state?

No, there isn't.

We can expose it, but do you think adding this API requires it (i.e. we can't have one without the other)?

Reply:
I think adding this API exposes a need for it.  Library authors wouldn't know that it's safe (or not) to use this method.  It's a precondition exception that they can't check.